great_expectations/checkpoint/configurator.py

```diff
@@ -165,7 +165,7 @@ def _build_checkpoint_config(self) -> CheckpointConfig:
 
         logger.debug(
             f"SimpleCheckpointConfigurator built this CheckpointConfig:"
-            f" {print(checkpoint_config)}"
+            f" {str(checkpoint_config)}"
```

Comment:
Do we even need to explicitly invoke `str()`? Don't f-strings do that conversion implicitly? 

Either way, I'm good 😄

Reply:
By default f-strings use `__str__()`

Reply:
Makes sense to me; no reason to make the extra call.